pyfr/integrators/controllers.py

```diff
@@ -49,6 +51,13 @@ def __init__(self, *args, **kwargs):
         # Delete the memory-intensive elements map from the system
         del self.system.ele_map
 
+    def ensure_tstep(self, tstep):
```

Comment:
OK. And just replace tstep with dt?

Peter

Dr Peter Vincent MSci ARCS DIC PhD
Senior Lecturer and EPSRC Early Career Fellow
Department of Aeronautics
Imperial College London
South Kensington
London
SW7 2AZ
UK

web: www.imperial.ac.uk/aeronautics/research/vincentlab<http://www.imperial.ac.uk/aeronautics/research/vincentlab>
twitter: @Vincent_Lab<https://twitter.com/Vincent_Lab>





On 1 Oct 2015, at 16:27, iyer-arvind <notifications@github.com<mailto:notifications@github.com>> wrote:


In pyfr/integrators/controllers.py<https://github.com/vincentlab/PyFR/pull/71#discussion_r40926736>:

> @@ -49,6 +51,13 @@ def __init__(self, *args, **kwargs):
>          # Delete the memory-intensive elements map from the system
>          del self.system.ele_map
>
> +    def ensure_tstep(self, tstep):


call_plugin_dt ?

—
Reply to this email directly or view it on GitHub<https://github.com/vincentlab/PyFR/pull/71/files#r40926736>.